test/Sentry.Tests/HubTests.cs

```diff
@@ -1376,4 +1376,32 @@ public async Task WithScopeAsyncT_Works()
     }
 
     private static Scope GetCurrentScope(Hub hub) => hub.ScopeManager.GetCurrent().Key;
+
+    [Fact]
+    public async Task IdleTimeout_auto_finishes_transaction()
```

Comment:
Does this test something we're not already testing in `TransactionTests.Finish_SentryRequestTransactionGetsIgnored`?

Reply:
Yes, it's redundant. Will remove.